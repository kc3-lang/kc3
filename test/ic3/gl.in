quote %GL.Vec2{}
%GL.Vec2{}
quote %GL.Vec3{}
%GL.Vec3{}
quote %GL.Vertex{}
%GL.Vertex{}
quote %GL.Triangle{}
%GL.Triangle{}
quote %GL.Object{}
%GL.Object{}
