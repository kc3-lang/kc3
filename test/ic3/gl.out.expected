%GL.Vec2{}
%GL.Vec2{x: 0.0f, y: 0.0f}
%GL.Vec3{}
%GL.Vec3{x: 0.0f, y: 0.0f, z: 0.0f}
%GL.Vertex{}
%GL.Vertex{position: %GL.Vec3{x: 0.0f, y: 0.0f, z: 0.0f}, normal: %GL.Vec3{x: 0.0f, y: 0.0f, z: 0.0f}, tex_coord: %GL.Vec2{x: 0.0f, y: 0.0f}}
%GL.Triangle{}
%GL.Triangle{a: 0, b: 1, c: 2}
%GL.Object{}
%GL.Object{vertex: (GL.Vertex[]) {}, triangle: (GL.Triangle[]) {}, gl_mode: (U32) 0, gl_vao: (U32) 0, gl_vbo: (U32) 0, gl_ebo: (U32) 0}
